app/src/main/java/edu/akelael/comics/model/IComicListModel.java

```diff
@@ -0,0 +1,6 @@
+package edu.akelael.comics.model;
+
+public interface IComicListModel {
```

Comment:
I think yes, but in this exercise it's difficult to distinguish the correct use of the interfaces...